tokio/src/sync/broadcast.rs

```diff
@@ -691,6 +691,70 @@ impl<T> Drop for Sender<T> {
 }
 
 impl<T> Receiver<T> {
+    /// Returns the number of messages that were sent into the channel and that
+    /// this [`Receiver`] has yet to receive.
+    ///
+    /// If the returned value from `len` is larger or equal to the capacity of
+    /// the channel any call to [`recv`] will return an `Err(RecvError::Lagged)`
+    /// and any call to [`try_recv`] will return an `Err(TryRecvError::Lagged)`.
```

Comment:
Do you also get a lagged error if they are equal? I don't think so.

Maybe a test would be in order.